com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerVisualizer.cs

```diff
@@ -48,7 +48,7 @@ public class ControllerVisualizer : MonoBehaviour
         /// Cached reference to hands aggregator for efficient per-frame use.
         /// </summary>
         [Obsolete("Deprecated, please use XRSubsystemHelpers.HandsAggregator instead.")]
-        protected HandsAggregatorSubsystem HandsAggregator => XRSubsystemHelpers.HandsAggregator;
+        protected HandsAggregatorSubsystem HandsAggregator => (HandsAggregatorSubsystem)XRSubsystemHelpers.HandsAggregator;
```

Comment:
Can this be changed to IHandsAggregatorSubsystem ?    XRSubsystemHelpers may hold something that is not a HandsAggregatorSubsystem.

If it can't be changed, use the "as" operator.